src/main/scala/who/ate/all/the/pies/SymbolType.scala

```diff
@@ -0,0 +1,5 @@
+package who.ate.all.the.pies
+
+case class SymbolType(v: Var, index: Int) extends Expr {
```

Comment:
I get what yer up to here, did you consider making use of https://www.scala-lang.org/api/2.13.6/scala/Symbol.html in any way?

Reply:
I see, so using this: 
case class SymbolType(v: Symbol, index: Int) extends Expr {